doc/11-Concepts.md

```diff
@@ -0,0 +1,77 @@
+
+### Meta
+The meta area is for additional information, usually displays of time. In the Host and Service Lists, you can see how long the object has been in the corresponding state.
+
+### Caption
+The caption area contains detailed information about the list item. In the case of hosts and services this would be the plugin output. For comments and downtimes the comment texts of the user are displayed here. To make the default list view compact and consistent, long texts are truncated to a maximum of two lines.
+
+## Different detail degrees
+
+The level of detail of the host and service lists is now selectable - the above image shows the default list item, but there are two more iterations:
+
+In the compact mode the entire list item is displayed in one single line.
+Given, that there is enough space, the caption is placed in the same line as the rest and trimmed to fit.
+![CompactAnatomy](res/ListItemAnatomyCompact.jpg "A more compact list item that fits in one line")
+
+The detailed mode displays the entire caption, which makes it possible to react to problems directly from the list view.
+![DetailedAnatomy](res/ListItemAnatomyDetailed.jpg "A more detailed list item that spans four lines")
+
+## Overdue Items
+
+In host and service lists overdue checks are now highlighted. This makes it immediately apparent which objects may no longer be up-to-date. The red badge contains information about how long the check has overdue.
+![OverdueListAnatomy](res/ListAnatomyOverdue.jpg "A list with a highlighted item")
+
+## Downtimes
+Downtime lists have an extra information to be displayed: the progress.
+How far along the downtime is, is now immeadiately visible via the progress bar at the top of each list item.
+In the only square visual, we display how much time is left until the downtime ends.
+![DowntimeListItem](res/ListItemDowntimeAnatomy.jpg "A downtime list item with the added progress bar in the top")
+
+
+# Modal
+If you wanted to create a comment or set a downtime from the detail area, the dialog was displayed in another column, which meant that the contents of the left column were lost. In the icingadb module, there is now a modal element for short interaction dialogues. For actions in the detail area, a modal dialog will now appear. This preserves the left list column and thus the context better.
+![ModalAnatomy](res/ModalAnatomy.jpg "A modal presenting a form")
+
+### Utility
+The utility or close button is one if the three ways to close the modal without confirming your changes. You may also just click outside it's borders or press the escape key to escape its' grasp.
+
+### Title
+The title tells you what is asked off you in a simple keyword. Usually an action like: Acknowledge Problem, Add Comment or Schedule Downtime.
```

Comment:
- You again.
- What is asked off me ... I don't understand that sentence. Since it refers to an action later, why not rephrase that to `The title specifies the target, e.g. an action like: ...`